manual/installation.tex

```diff
@@ -196,71 +203,89 @@ \subsection{Configuration options}
-QMC_MIXED_PRECISION Build the mixed precision (mixing double/float) version
-                    (1:yes (GPU default), 0:no (CPU default)).
-                    The CPU support is experimental.
-                    Use float and double for base and full precision.
-                    The GPU support is quite mature.
-                    Use always double for host side base and full precision
-                    and use float and double for CUDA base and full precision.
-ENABLE_TIMERS       Enable fine-grained timers (1:yes, 0:no (default)).
-                    Timers are off by default to avoid potential slowdown in small
-                    systems. For large systems (100+ electrons) there is no risk.
-ENABLE_SOA          (Experimental) Enable CPU optimization based on Structure-
-                    of-Array (SoA) datatypes (1:yes, 0:no (default)). 
-\end{verbatim}
-  \item General build options
-\begin{verbatim}
-CMAKE_BUILD_TYPE   A variable which controls the type of build
-                   (defaults to Release). Possible values are:
-                   None (Do not set debug/optmize flags, use
-                   CMAKE_C_FLAGS or CMAKE_CXX_FLAGS)
-                   Debug (create a debug build)
-                   Release (create a release/optimized build)
-                   RelWithDebInfo (create a release/optimized build with debug info)
-                   MinSizeRel (create an executable optimized for size)
-CMAKE_C_COMPILER   Set the C compiler
-CMAKE_CXX_COMPILER Set the C++ compiler
-CMAKE_C_FLAGS      Set the C flags.  Note: to prevent default
-                   debug/release flags from being used, set the CMAKE_BUILD_TYPE=None
-                   Also supported: CMAKE_C_FLAGS_DEBUG,
-                   CMAKE_C_FLAGS_RELEASE, and CMAKE_C_FLAGS_RELWITHDEBINFO
-CMAKE_CXX_FLAGS    Set the C++ flags.  Note: to prevent default
-                   debug/release flags from being used, set the CMAKE_BUILD_TYPE=None
-                   Also supported: CMAKE_CXX_FLAGS_DEBUG,
-                   CMAKE_CXX_FLAGS_RELEASE, and CMAKE_CXX_FLAGS_RELWITHDEBINFO
-\end{verbatim}
+\verbatimfont{\footnotesize}%
+\begin{verbatim}
+QMC_CUDA              Enable CUDA and GPU acceleration (1:yes, 0:no)
+QMC_COMPLEX           Build the complex (general twist/k-point) version (1:yes, 0:no)
+QMC_MIXED_PRECISION   Build the mixed precision (mixing double/float) version
+                      (1:yes (GPU default), 0:no (CPU default)).
+                      The CPU support is experimental.
+                      Use float and double for base and full precision.
+                      The GPU support is quite mature.
+                      Use always double for host side base and full precision
+                      and use float and double for CUDA base and full precision.
+ENABLE_MKL            Enable Intel MKL libraries.
+MKL_ROOT              Path to MKL libraries (only necessary for non intel compilers
```

Comment:
ENABLE_MKL and MKL_ROOT are not key build options. Please list them under addition build options